pkg/common/common.go

```diff
@@ -146,7 +146,11 @@ const (
 
 	PartitionDiskQuotaKey = "partition.diskProtection.diskQuota.mb"
 
-	// database level properties
+	// collection level load properties
+	CollectionReplicaNumber  = "collection.replica.number"
+	CollectionResourceGroups = "collection.resource_groups"
```

Comment:
camel case?

Reply:
maybe a bit ugly, but keep up with the database level configs